addons/elasticsearch/es-search-logs-mappings.json

```diff
@@ -0,0 +1,75 @@
+{
+  "mappings": {
+    "properties": {
+      "request.attributes": {
+        "type": "keyword"
+      },
+      "request.relationAttributes": {
+        "type": "keyword"
+      },
+
+      "userAgent": {
+        "type": "keyword"
+      },
+      "host": {
+        "type": "keyword"
+      },
+      "ipAddress": {
+        "type": "keyword"
+      },
+      "userName": {
+        "type": "keyword"
+      },
+
+
+      "entityGuidsAll": {
+        "type": "keyword"
+      },
+      "entityQFNamesAll": {
+        "type": "keyword"
+      },
+      "entityGuidsAllowed": {
+        "type": "keyword"
+      },
+      "entityQFNamesAllowed": {
+        "type": "keyword"
+      },
+      "entityGuidsDenied": {
+        "type": "keyword"
+      },
+      "entityQFNamesDenied": {
+        "type": "keyword"
+      },
+      "utmTags": {
+        "type": "keyword"
+      },
+
+      "hasResult": {
+        "type": "boolean"
+      },
+      "isFailed": {
```

Comment:
`isFailed` represents what ? Does this infer request failed ?

Reply:
Yes, it tells whether indexsearch request was failed or successful